src/urbit/lib/chess.hoon

```diff
@@ -1394,6 +1395,33 @@
       (algebraicize-and-number game)
     ==
 ::
+::  helps chess-archive mark
+++  archive-json
+  |=  archive=[@dau chess-game]
+  =/  game  +.archive
```

Comment:
This is the extra logic added for the `archive` as `mop` that I'm not sure would transfer to the `%chess-game` mark.

In PR #109, I bring it up in [this comment](https://github.com/thecommons-urbit/chess/pull/109#discussion_r1093451608)

Reply:
Based on my thoughts in #65, this will need to be slightly different: we should send the metadata info but not the moves when sending the archive.

I'd also prefer it to remain in the `mark` file. I don't think there's anything incorrect  about how you chose to do it, it just seems to be a convention that mark files go to great lengths to be self-contained. It's also a very standard JSON conversion, so I don't think it warrants being in the `lib`.